src/components/App.tsx

```diff
@@ -1587,17 +1587,21 @@ class App extends React.Component<AppProps, AppState> {
       }
 
       if (event.key === KEYS.G || event.key === KEYS.S) {
-        if (this.state.elementType === "selection") {
+        const selectedElements = getSelectedElements(
+          this.scene.getElements(),
+          this.state,
+        );
+        if (
+          this.state.elementType === "selection" &&
+          selectedElements.length === 0
```

Comment:
```suggestion
          !selectedElements.length
```

I believe this was GH Copilot's fault :)

Reply:
I actually like === 0 better :p